packages/react-composites/tests/browser/call/hermetic/RoomsCallScreen.test.ts

```diff
@@ -71,3 +79,86 @@ test.describe('Rooms CallScreen tests for different roles', async () => {
     );
   });
 });
+
+test.describe('Rooms Participant RemoveButton tests for different roles', async () => {
+  test('Remove button is enabled for Presenter', async ({ page, serverUrl }, testInfo) => {
+    test.skip(isTestProfileStableFlavor());
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
+    const participants = [paul];
+    const initialState = defaultMockCallAdapterState(participants);
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { role: 'Presenter' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+
+    if (!isTestProfileDesktop(testInfo)) {
```

Comment:
nit: Avoid negation here. The `true` branch can be for desktop instead of the other way around.

Reply:
done